crates/meilisearch/src/extractors/authentication/mod.rs

```diff
@@ -214,14 +214,14 @@ pub mod policies {
         Ok(api_key_uid)
     }
 
-    fn is_keys_action(action: u8) -> bool {
+    fn is_keys_action(action: u32) -> bool {
         use actions::*;
         matches!(action, KEYS_GET | KEYS_CREATE | KEYS_UPDATE | KEYS_DELETE)
     }
 
-    pub struct ActionPolicy<const A: u8>;
+    pub struct ActionPolicy<const A: u32>;
 
-    impl<const A: u8> Policy for ActionPolicy<A> {
+    impl<const A: u32> Policy for ActionPolicy<A> {
```

Comment:
Wouldn't it be better to be explicit about what we expect, here, and use the `Action` type instead of the `u32` one?